android/java/org/chromium/chrome/browser/local_database/DisplayAdsTable.java

```diff
@@ -0,0 +1,138 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.local_database;
+
+public class DisplayAdsTable {
+    public static final String TABLE_NAME = "display_ads_table";
+
+    public static final String COLUMN_UUID = "uuid";
+    public static final String COLUMN_CREATIVE_INSTANCE_ID = "creative_instance_id";
+    public static final String COLUMN_POSITION = "position";
+    public static final String COLUMN_TAB_ID = "tab_id";
+    public static final String COLUMN_AD_TITLE = "ad_title";
+    public static final String COLUMN_AD_DESCRIPTION = "ad_description";
+    public static final String COLUMN_AD_CTA_TEXT = "cta_text";
+    public static final String COLUMN_AD_CTA_LINK = "cta_link";
+    public static final String COLUMN_AD_IMAGE = "ad_image";
+
+    // Create table SQL query
+    public static final String CREATE_TABLE = "CREATE TABLE IF NOT EXISTS " + TABLE_NAME
+            + "( ID INTEGER PRIMARY KEY AUTOINCREMENT," + COLUMN_UUID + " TEXT,"
+            + COLUMN_CREATIVE_INSTANCE_ID + " TEXT," + COLUMN_POSITION + " INT," + COLUMN_TAB_ID
+            + " INT," + COLUMN_AD_TITLE + " TEXT," + COLUMN_AD_DESCRIPTION + " TEXT,"
+            + COLUMN_AD_CTA_TEXT + " TEXT," + COLUMN_AD_CTA_LINK + " TEXT," + COLUMN_AD_IMAGE
+            + " TEXT"
+            + ")";
+
+    public DisplayAdsTable() {}
+
+    private String uuid;
```

Comment:
private members naming

Reply:
@AlexeyBarabash yeah i guess i should follow the other similar class conventions. Changing... But I don't like using mXYZ for custom type classes esp. when you know there is no more complexity then setters and getters.

Reply:
Sorry for that, but we should follow Chromium code style guide. Thanks for changes :+1: